KPI/kpi.model.lkml

```diff
@@ -0,0 +1,21 @@
+connection: "telemetry"
+
+include: "./dashboards/*.dashboard"
+include: "./views/*.view.lkml"                # include all views in the views/ folder in this project
+
+explore: firefox_desktop_usage_2021 {
+  label: "Firefox Desktop Usage"
+  group_label: "KPIs"
+  from: firefox_desktop_usage_2021
+  join: prediction {
+    view_label: "Forecast"
+    type: left_outer
+    sql_on: ${firefox_desktop_usage_2021.date} = ${prediction.date} ;;
+    relationship: one_to_one
+  }
+}
+
+# For suggestions
+explore: firefox_desktop_usage_fields {
```

Comment:
Because we have a sql derived_table query for `firefox_desktop_usage_2021` (see e.g. `select submission_date, sum(dau) as dau, ...`), we don't get all the dimension values from that query, so Looker can't use it for the filter suggestions.

Instead we create a parallel, hidden explore that we use for suggestions on those filter fields.